content/SCALE/SCALEUIReference/Apps/AppsScreensSCALE.md

```diff
@@ -216,11 +222,11 @@ The **Available Applications** screen displays the widgets for all applications
 
 The **Install** button on each application card opens the configuration wizard for that application.
 
-Click on the application icon or name to open an ***appname* Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status** and **Versions** for that application.
+Click on the application icon or name to open an **appname** Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status**, and **Versions** for that application.
```

Comment:
The bold emphasis is off here, what part of this is bold and on the UI screen? if one is a variable and the other is static on the UI screen, then make this change
```suggestion
Click on the application icon or name to open an ***appname* Application Summary** window that includes information on the **Catalog**, **Categories**, **Train**, **Status**, and **Versions** for that application.
```